app/packages/klogs/src/components/page/LogsRowPreview.tsx

```diff
@@ -0,0 +1,55 @@
+import { TableCell, Typography, Box } from '@mui/material';
+import { grey } from '@mui/material/colors';
+import { alpha } from '@mui/material/styles';
+import { Fragment, FunctionComponent } from 'react';
+
+import { IRow } from '../common/types';
+
+interface IRowPreviewProps {
+  row: IRow;
+}
+
+/**
+ * RowPreview renders a preview of a log document with pre-defined fields
+ */
+const RowPreview: FunctionComponent<IRowPreviewProps> = ({ row }) => {
+  const knownColumns = ['cluster', 'namespace', 'app', 'pod_name', 'container_name', 'host'];
+  const contentColumns = Object.keys(row).filter((col) => col.startsWith('content_'));
+
+  return (
+    <TableCell size="small">
+      <Typography
```

Comment:
I wouldn't cap this to two lines and instead use the existing logic to display some more information